multinet/multinet/resolvers/query.py

```diff
@@ -5,11 +5,14 @@
 
 def workspaces(root, info, name=""):
     """Return list of workspace names accessible to the user."""
-    return [
-        workspace
-        for workspace in db.get_workspaces(name)
-        if not name or workspace == name
-    ]
+    if db.get_workspaces(name) is not None:
+        return [
+            workspace
+            for workspace in db.get_workspaces(name)
+            if not name or workspace == name
+        ]
+    else:
+        return
```

Comment:
This would be the place to raise an error if the database is not live. The way you have it now, you would want to replace this with something like `return (None, "500 Database Not Running")`.

Reply:
We decided to move the error handling to the multinet.py file